requirements-dev.txt

```diff
@@ -1,6 +1,9 @@
 pytest==5.2.1
 flake8==7.0.0
-autopep8==1.4.4
+autopep8==2.0.4
 pytest-mock==3.7.0
-coverage==6.3.1
-pyyaml==6.0
\ No newline at end of file
+coverage==7.6.4
+pyyaml==6.0.2
+anywidget==0.9.13
+starlette==0.41.0
+uvicorn==0.32.0
```

Comment:
These optional vitessce dependencies throw warnings but don't otherwise cause any issues - I think it'd be better to keep these out to avoid causing issues downstream where these dependencies aren't needed (e.g. search api/portal UI don't really care about starlette since it's only relevant for local data hosting/don't care about anywidget since that's mainly just for jupyter)

Reply:
thats a good point, will remove these